sky/clouds/oci.py

```diff
@@ -490,3 +492,8 @@ def _get_default_image(self, region_name: str, instance_type: str) -> str:
         raise exceptions.ResourcesUnavailableError(
             'ERR: No image found in catalog for region '
             f'{region_name}. Try update your default image_id settings.')
+
+    @classmethod
+    def check_disk_tier_enabled(cls, instance_type: str,
+                                disk_tier: str) -> None:
+        del instance_type, disk_tier  # unused
```

Comment:
```suggestion
        # All the disk_tier are supported for any instance_type
        del instance_type, disk_tier  # unused
```
(is this true?)

Reply:
Yes, though some shapes may not be able to benifit ultra high performance vpu setting. This is same with the operation via OCI console, which does not limit the user to select a higher vpu value, so here we do nothing specific constraints as well.